conda/core/initialize.py

```diff
@@ -852,7 +852,7 @@ def run_plan_elevated(plan):
                     rm_rf(temp_path)
 
         else:
-            stdin = json.dumps(plan, ensure_ascii=False, default=lambda x: x.__dict__)
+            stdin = json.dumps(plan, ensure_ascii=False, default=lambda x: x.__dict__).encode('utf-8')
```

Comment:
Minor reformat to make pre-commit/ruff happy:

```suggestion
            stdin = json.dumps(
                plan,
                ensure_ascii=False,
                default=lambda x: x.__dict__,
            ).encode('utf-8')
```